libs/mark-flow-ui/src/pages/start_page.tsx

```diff
@@ -52,12 +50,6 @@ const Footer = styled.div`
   padding: 0.5rem;
 `;
 
-const LargeButtonText = styled.span`
-  font-size: 50px;
-  line-height: 2;
-  padding: 0 1rem;
-`;
-
```

Comment:
I am confused by the styling change here:

1. Was this button really a set `px` before, and thus unresponsive to size settings?
2. From my reading, it looks like you made it responsive but a lot smaller, from `50px` to `1.2rem`? Is that right?

Reply:
1. Yes, as far as I can tell it was a set px before.
2. The font size with the default medium size theme now works out to 48px, so it's pretty much in the same range.